services/comps/virtio/src/device/block/device.rs

```diff
@@ -37,14 +38,20 @@ pub struct BlockDevice {
 impl BlockDevice {
     /// Creates a new VirtIO-Block driver and registers it.
     pub(crate) fn init(transport: Box<dyn VirtioTransport>) -> Result<(), VirtioDeviceError> {
+        let name = transport.get_device_name();
+
         let block_device = {
             let device = DeviceInner::init(transport)?;
             Self {
                 device,
                 queue: BioRequestSingleQueue::new(),
             }
         };
-        aster_block::register_device(super::DEVICE_NAME.to_string(), Arc::new(block_device));
+
+        aster_block::register_device(
```

Comment:
Add comments to explain the format of the name

Reply:
Fixed, the name is now specified in QEMU arguments by `--serial`.